src/sql/src/session/vars.rs

```diff
@@ -5056,6 +5110,33 @@ impl Value for Vec<Ident> {
     }
 }
 
+impl Value for Vec<SerializableDirective> {
+    fn type_name() -> String {
+        "directive list".to_string()
+    }
+
+    fn parse<'a>(
+        param: &'a (dyn Var + Send + Sync),
+        input: VarInput,
+    ) -> Result<Vec<SerializableDirective>, VarError> {
+        let values = input.to_vec();
+        let dirs: Result<_, _> = values
+            .iter()
+            .flat_map(|i| i.split(','))
+            .map(|d| SerializableDirective::from_str(d.trim()))
+            .collect();
+        dirs.map_err(|e| VarError::InvalidParameterValue {
+            parameter: param.into(),
+            values: values.to_vec(),
+            reason: e.to_string(),
+        })
+    }
+
+    fn format(&self) -> String {
+        self.iter().map(|d| d.to_string()).join(", ")
+    }
+}
```

Comment:
Not covered by any tests according to the coverage report. Could you please add a test here?